CMakeLists.txt

```diff
@@ -5,7 +5,10 @@
 cmake_minimum_required(VERSION 3.1)
 project(TYPE_SAFE)
 
-include(external/external.cmake)
+find_package(debug_assert)
+if(NOT debug_assert_FOUND)
+    include(external/external.cmake)
```

Comment:
Oh, and again: Is ist still version 3.1 or does the cmake_minimum_required() needs an update?